types/three/three-dragcontrols.d.ts

```diff
@@ -0,0 +1,5 @@
+import {Camera} from './three-core';
+
+export class DragControls {
+    constructor(objects: any, camera: Camera, domElement?: HTMLElement);
```

Comment:
Is `objects` supposed to be `any`?

Reply:
Not 100% on this one, but I think `objects` expects `Object3d`

Reply:
Looking into the raycaster types (used by DragControls), it looks like this should be an array, so `Object3D[]`. I'll make the update.